README.md

```diff
@@ -3,6 +3,7 @@
 [![Coverage Status](https://coveralls.io/repos/github/publiclab/plotsbot/badge.svg?branch=master)](https://coveralls.io/github/publiclab/plotsbot?branch=master)
 [![license](https://img.shields.io/badge/license-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0)
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](http://makeapullrequest.com/)
+[![Dependencies](https://img.shields.io/versioneye/d/ruby/rails.svg)](https://github.com/publiclab/plotsbot#dependencies)
```

Comment:
Thanks @bassdeveloper for handling this so well, and @prabeous73 for complying properly.